docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,335 @@
+```csharp
+window.Resized += OnWindowResized;
+
+private void OnWindowResized(object sender, Window.ResizedEventArgs e)
+{
+    // Window is resized, do something
+}
+```
+
+<a name="Rotation"></a>
+## Window Rotation
+
+To support rotation, NUI provides two methods.
+
+### Enable/Disable rotation
+
+If the device supports rotation, application can rotate the window with specific orientation.
+
+The supported orientation are Portriat, Landscape, PortraitInverse and LandscapeInverse.
+Portrait means the width is larger than the height or same.
+Otherwise Landscape means the width is larger than the height.
+
+<img src="./media/device_rotate2.png">
+
+
+Application can enable or disable the specifc orientation to use orientation functions. The below example means that Portrait, Landscape and LandscapeInverse are supported in the application.
+If the device's width is larger than the height, the window will be shown with Landscape when application is lanunched. Otherwise the device's width is larger than the height or same, the window will be shown with Portrait.   
+
+```csharp
+// To enable Portrait
+window.AddAvailableOrientation( Portrait );
+
+// To enable Landscape
+window.AddAvailableOrientation( Landscape );
+
+// To disable PortraitInverse
+window.RemoveAvailableOrientation( PortraitInverse );
+
+// To enable LandscapeInverse
+window.AddAvailableOrientation( LandscapeInverse );
+```
+
+
+### Preferred orientation
+
+Sometimes, the window should only work the specific orientation.
+For the case, the preferred orientation function is supplied.
+We recommand that the preferred orientation is one of avialable orientations.
+For instance, look at the below example.
+The available orientation list has Portrait, Landscape and LandscapeInverse, then the preferred orientation is set with Landscape.
+After that, the window will only work Landscape although device is rotated with any orientation.
+
+```csharp
+// To enable available orientations
+window.AddAvailableOrientation( Portrait );
+window.AddAvailableOrientation( Landscape );
+window.AddAvailableOrientation( LandscapeInverse );
+
+// To set the preferred orientation with Landscape.
+window.SetPreferredOrientation( Landscape );
+```                                                                                                                               
+
+> **Note**
+>
+> If the Preferred Orientation's function is used without the available orientation list, the rotation will not work normal.
+>
+
+<a name="multiwindow"></a>
+## Multi Window
+
+<a name="additionalwindow"></a>
+### Creating Additional Window
+
+It is easy to create an additional window in addition to the default window. The size and position of the new window should be specified. Each window can has its own background color and title.
+
+```csharp
+Window newWindow = new Window(new Rectangle(0, 0, 1920, 1080))
+{
+    BackgroundColor = Color.White,
+    Title = "new window"
+};
+```
+
+<img src="./media/multiwindow3.png">
```

Comment:
I can see the image and the sample code doesn't match.
It would be better if you can use the sample code for that image.